### PR TITLE
Run sync cluster resources as a goroutine in sandbox

### PR DESCRIPTION
**Pull request by katrogan**
Signed-off-by: Katrina Rogan <katroganGH@gmail.com>

https://github.com/flyteorg/flyte/issues/162

### CODE REVIEW

File: deployment/eks/flyte_generated.yaml [hunk 1]
```diff
@@ -8121,7 +8121,7 @@ data:
                 value: "4"
             - projectQuotaMemory:
                 value: "3000Mi"
-      refresh: 5m
+      refreshInterval: 5m
```
Comment (kumare3) [inline deployment/eks/flyte_generated.yaml:8124]: can we make this 1 minute?

Comment (katrogan) [reply deployment/eks/flyte_generated.yaml]: updated

**Comment by kumare3** [comment]
@katrogan end to end tests are failing

**Comment by katrogan** [comment]
@kumare3 it's still failing the helm chart diff. is there a command that I'm missing outside of `make helm` ?

**Comment by katrogan** [comment]
PTAL @kumare3

**Review by kumare3 — APPROVED** [review]

**Pull request merged**